Rehlate/index.html

```diff
@@ -0,0 +1,373 @@
+            <p>كونك شخص ذو ذوق خاص ستكون هذه الخطة هي ما يناسبك تماماً</p>
+            <hr />
+            <ul>
+              <li>بطاقات سفر بدرجة رجال الأعمال</li>
+              <li>حجز بفندق خمس نجوم</li>
+              <li>متابعة خاصة لإحتياجاتك</li>
+            </ul>
+            <button>
+                احجز الآن
+                <img src="./assets/navigators/arrow-left.svg" />
+              </button>
+          </div>
+        </div>
+      </section>
+
+      <section class="Reviews">
+        <h4 class="heading-grey">آراء العملاء</h4>
+        <h2 class="black">ماذا يقولون عنا</h2>
+        <div class="reviews-cards">
+          <div class="review-card card-bg-grey-bg">
+            <img src="./assets/SVG/Stars.svg" alt="review" />
+            <div class="inline">
+            <img src="./assets/SVG/reviews/review-01.svg" alt="review-01" />
+            <h5>محمد الأحمد</h5>
+            </div>
+            <p>
+              كانت تجربة مميزة مع خدمات رائعة قضيت أفضل إجازة لي من فترة طويلة،
+              أنصح بالتعامل مع شركة رحلاتي بدون تردد.
+            </p>
+          </div>
+          <div class="review-card card-bg-grey-bg">
+            <img src="./assets/SVG/Stars.svg" alt="review" />
+            <div class="inline">
+            <img src="./assets/SVG/reviews/review-02.svg" alt="review-02" />
+            <h5>ماهر عبد اللطيف</h5>
+            </div>
+            <p>
+              تقدم شركة رحلاتي خدمات مناسبة جداً لرجال الأعمال، هذا ما ساعدني في
+              رحلتي وجعلني اختصر جداً من مشاكل الحجوزات.
+            </p>
+          </div>
+          <div class="review-card card-bg-grey-bg">
+            <img src="./assets/SVG/Stars.svg" alt="review" />
+            <div class="inline">
+            <img src="./assets/SVG/reviews/review-03.svg" alt="review-03" />
+            <h5>أسامة الأبيض</h5>
+            </div>
+            <p>
+              يمكنني القول إنني قمت بقطاء أفضل رحلة عائلة على الإطلاق، كان
+              الفنادق رائعة وخطة السفر ممتازة جداً ومريحة.
+            </p>
+          </div>
+        </div>
+        <div class="desktop">
+        <button>
+          <img src="./assets/navigators/arrow-left-purple.svg" />
+        </button>
+        <button >
+          <img src="./assets/navigators/arrow-right-purple.svg" />
+        </button>
+        </div>
+      </section>
+
+      <section class="contact">
+        <h4 class="heading-grey">هل لديك سؤال؟</h4>
+        <h2 class="black">يسعدنا التواصل معك</h2>
+        <div>
+          <div class="contact-form">
+            <form class="pink-bg">
+                <span>
+              <label>الاسم</label>
+              <input type="text" placeholder="يرجى تسجيل اسمك الكامل" />
+                </span>
+            <span>
+              <label>البريد الإلكتروني</label>
+              <input type="email" placeholder="سجل بريدك الشخصي" />
+            </span>
+            <span>
+              <label>الموضوع</label>
+              <textarea
```

Comment:
Please increase the space in this t`ext area`. You can adjust the` cols `and `rows `**attributes** in the `<textarea>` to match the design